Wheel tests failing on WKT CRS tests

I am wondering if this is a PROJ version issue where older pyproj is needed on earlier python versions and they cause the parsing to fail

```
2026-06-18T21:16:14.3270501Z D:\a\sedona-db\sedona-db\python\sedonadb\tests\test_udf.py::test_udf_matchers PASSED [ 99%]
2026-06-18T21:16:14.3298479Z D:\a\sedona-db\sedona-db\python\sedonadb\tests\test_udf.py::test_udf_types PASSED [ 99%]
2026-06-18T21:16:14.3325268Z D:\a\sedona-db\sedona-db\python\sedonadb\tests\test_udf.py::test_udf_any_input PASSED [ 99%]
2026-06-18T21:16:14.3350666Z D:\a\sedona-db\sedona-db\python\sedonadb\tests\test_udf.py::test_udf_return_type_fn PASSED [ 99%]
2026-06-18T21:16:14.3377357Z D:\a\sedona-db\sedona-db\python\sedonadb\tests\test_udf.py::test_udf_array_input PASSED [ 99%]
2026-06-18T21:16:14.3388295Z D:\a\sedona-db\sedona-db\python\sedonadb\tests\test_udf.py::test_udf_name PASSED [ 99%]
2026-06-18T21:16:14.3521103Z D:\a\sedona-db\sedona-db\python\sedonadb\tests\test_udf.py::test_shapely_udf PASSED [ 99%]
2026-06-18T21:16:14.3539975Z D:\a\sedona-db\sedona-db\python\sedonadb\tests\test_udf.py::test_py_sedona_value PASSED [ 99%]
2026-06-18T21:16:14.3570153Z D:\a\sedona-db\sedona-db\python\sedonadb\tests\test_udf.py::test_udf_kwargs PASSED [ 99%]
2026-06-18T21:16:14.3589960Z D:\a\sedona-db\sedona-db\python\sedonadb\tests\test_udf.py::test_udf_bad_return_object PASSED [ 99%]
2026-06-18T21:16:14.3611113Z D:\a\sedona-db\sedona-db\python\sedonadb\tests\test_udf.py::test_udf_bad_return_type PASSED [ 99%]
2026-06-18T21:16:14.6303268Z D:\a\sedona-db\sedona-db\python\sedonadb\tests\test_udf.py::test_udf_bad_return_length PASSED [100%]
2026-06-18T21:16:14.6304119Z 
2026-06-18T21:16:14.6304302Z ================================== FAILURES ===================================
2026-06-18T21:16:14.6304784Z _______________________ test_st_transform_from_wkt_crs ________________________
2026-06-18T21:16:14.6305091Z 
2026-06-18T21:16:14.6305324Z con = <sedonadb.context.SedonaContext object at 0x000002D0CA02B880>
2026-06-18T21:16:14.6305635Z 
2026-06-18T21:16:14.6305785Z     def test_st_transform_from_wkt_crs(con):
2026-06-18T21:16:14.6306244Z         """A WKT CRS feeds the PROJ transform like any other CRS string: a point at
2026-06-18T21:16:14.6306829Z         the EPSG:3857 coordinates of (lon 1, lat 1) transforms back to ~POINT (1 1)."""
2026-06-18T21:16:14.6307280Z >       df = con.sql(
2026-06-18T21:16:14.6307544Z             "SELECT ST_Transform("
2026-06-18T21:16:14.6307968Z             "ST_SetCrs(ST_GeomFromText('POINT (111319.490793274 111325.142866385)'), $1),"
2026-06-18T21:16:14.6308417Z             " 'EPSG:4326') AS geom",
2026-06-18T21:16:14.6308705Z             params=(WKT_3857,),
2026-06-18T21:16:14.6308968Z         ).to_pandas()
2026-06-18T21:16:14.6309117Z 
2026-06-18T21:16:14.6309377Z D:\a\sedona-db\sedona-db\python\sedonadb\tests\functions\test_transforms.py:253: 
2026-06-18T21:16:14.6309883Z _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
2026-06-18T21:16:14.6310673Z ..\venv-test\lib\site-packages\sedonadb\dataframe.py:1186: in to_pandas
2026-06-18T21:16:14.6311132Z     table = self.to_arrow_table()
2026-06-18T21:16:14.6311479Z _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
2026-06-18T21:16:14.6311749Z 
2026-06-18T21:16:14.6311965Z self = <sedonadb.dataframe.DataFrame object at 0x000002D0D45323A0>
2026-06-18T21:16:14.6312357Z schema = None
2026-06-18T21:16:14.6312487Z 
2026-06-18T21:16:14.6312676Z     def to_arrow_table(self, schema: Any = None) -> "pa.Table":
2026-06-18T21:16:14.6321260Z         """Execute and collect results as a PyArrow Table
2026-06-18T21:16:14.6321763Z     
2026-06-18T21:16:14.6322147Z         Executes the logical plan represented by this object and returns a
2026-06-18T21:16:14.6322797Z         PyArrow Table. This requires that pyarrow is installed.
2026-06-18T21:16:14.6323234Z     
2026-06-18T21:16:14.6323447Z         Args:
2026-06-18T21:16:14.6323867Z             schema: The requested output schema or `None` to use the inferred
2026-06-18T21:16:14.6324326Z                 schema.
2026-06-18T21:16:14.6324560Z     
2026-06-18T21:16:14.6324757Z         Examples:
2026-06-18T21:16:14.6324973Z     
2026-06-18T21:16:14.6325207Z             >>> sd = sedona.db.connect()
2026-06-18T21:16:14.6325602Z             >>> sd.sql("SELECT ST_Point(0, 1) as geometry").to_arrow_table()
2026-06-18T21:16:14.6325987Z             pyarrow.Table
2026-06-18T21:16:14.6326327Z             geometry: extension<geoarrow.wkb<WkbType>> not null
2026-06-18T21:16:14.6326682Z             ----
2026-06-18T21:16:14.6326988Z             geometry: [[01010000000000000000000000000000000000F03F]]
2026-06-18T21:16:14.6327331Z     
2026-06-18T21:16:14.6327521Z         """
2026-06-18T21:16:14.6327758Z         import pyarrow as pa
2026-06-18T21:16:14.6328144Z         from sedonadb.utility import register_pyarrow_extension_types
2026-06-18T21:16:14.6328527Z     
2026-06-18T21:16:14.6328754Z         register_pyarrow_extension_types()
2026-06-18T21:16:14.6329039Z     
2026-06-18T21:16:14.6329373Z         # Collects all batches into an object that exposes __arrow_c_stream__()
2026-06-18T21:16:14.6329839Z >       batches = self._impl.to_batches(schema)
2026-06-18T21:16:14.6331156Z E       sedonadb._lib.SedonaError: Failed to create CRS from source 'PROJCS["WGS 84 / Pseudo-Mercator",GEOGCS["WGS 84",DATUM["WGS_1984",SPHEROID["WGS 84",6378137,298.257223563,AUTHORITY["EPSG","7030"]],AUTHORITY["EPSG","6326"]],AUTHORITY["EPSG","4326"]],PROJECTION["Mercator_1SP"],AUTHORITY["EPSG","3857"]]': Invalid PROJ string syntax
```

--- a/python/sedonadb/tests/functions/test_raster_functions.py
+++ b/python/sedonadb/tests/functions/test_raster_functions.py
@@ -46,8 +46,12 @@ def test_rs_function(expr, expected):
 WKT_3857 = (
     'PROJCS["WGS 84 / Pseudo-Mercator",GEOGCS["WGS 84",DATUM["WGS_1984",'
     'SPHEROID["WGS 84",6378137,298.257223563,AUTHORITY["EPSG","7030"]],'
-    'AUTHORITY["EPSG","6326"]],AUTHORITY["EPSG","4326"]],'
-    'PROJECTION["Mercator_1SP"],AUTHORITY["EPSG","3857"]]'
+    'AUTHORITY["EPSG","6326"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],'
+    'UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],'
+    'AUTHORITY["EPSG","4326"]],PROJECTION["Mercator_1SP"],'
+    'PARAMETER["central_meridian",0],PARAMETER["scale_factor",1],'
+    'PARAMETER["false_easting",0],PARAMETER["false_northing",0],'
+    'UNIT["metre",1,AUTHORITY["EPSG","9001"]],AUTHORITY["EPSG","3857"]]'
 )
 WKT_LCC_NO_AUTHORITY = (
     'PROJCS["Custom LCC",GEOGCS["WGS 84",DATUM["WGS_1984",'

--- a/python/sedonadb/tests/functions/test_transforms.py
+++ b/python/sedonadb/tests/functions/test_transforms.py
@@ -205,8 +205,12 @@ def test_st_crs_sedonadb(eng):
 WKT_3857 = (
     'PROJCS["WGS 84 / Pseudo-Mercator",GEOGCS["WGS 84",DATUM["WGS_1984",'
     'SPHEROID["WGS 84",6378137,298.257223563,AUTHORITY["EPSG","7030"]],'
-    'AUTHORITY["EPSG","6326"]],AUTHORITY["EPSG","4326"]],'
-    'PROJECTION["Mercator_1SP"],AUTHORITY["EPSG","3857"]]'
+    'AUTHORITY["EPSG","6326"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],'
+    'UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],'
+    'AUTHORITY["EPSG","4326"]],PROJECTION["Mercator_1SP"],'
+    'PARAMETER["central_meridian",0],PARAMETER["scale_factor",1],'
+    'PARAMETER["false_easting",0],PARAMETER["false_northing",0],'
+    'UNIT["metre",1,AUTHORITY["EPSG","9001"]],AUTHORITY["EPSG","3857"]]'
 )
 WKT_LCC_NO_AUTHORITY = (
     'PROJCS["Custom LCC",GEOGCS["WGS 84",DATUM["WGS_1984",'

